modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -18,63 +18,67 @@ endif::[]
 |===
 |Name |Parameter |Description
 
-ifdef::virtualmachine[]
+ifdef::vmtemplate[]
 |Template
 |
 |Template from which to create the virtual machine. Selecting a template will automatically complete other fields.
-endif::[]
-
-.4+|Source
-|PXE
-|Provision virtual machine from PXE menu. Requires a PXE-capable NIC in the cluster.
 
-|URL
-|Provision virtual machine from an image available from an *HTTP* or *S3* endpoint.
+|Name
+|
+|The name can contain lowercase letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (`.`), or special characters.
 
-|Container
-|Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`.
+|Template provider
```

Comment:
This field only available in template wizard, not availale in virtual machine wizard.

Reply:
High-performance feature was added back. Please note this file won't show up in this PR build because I removed it from the assembly, it pertains to a different PR that is focused just on virtual machines UI.